Lib/plistlib.py

```diff
@@ -508,12 +508,31 @@ def _get_size(self, tokenL):
 
         return tokenL
 
+    def _read(self, size):
+        if 0:
+            data = self._fp.read(size)
+            if len(data) != size:
+                raise InvalidFileException
+            return data
+        cursize = min(size, 1 << 20)
```

Comment:
Do you want to perhaps make (1 << 20) as a constant the same way you did for the pickle vulnerability?